### ISSUE TITLE
Data disappears and reappears randomly in edit texts

### ISSUE THREAD

**Issue by nathanRamaNoodles** [airbnb/epoxy#599]
First off, this library is amazing.  I've built some pretty slick UI's. 
Unfortunately, I've run into a very annoying bug. 

      Apparently, if you add too many edit text views, the controller starts to delete or clone the data from other edit texts.  What's even more weird is that all the stable id's are different. (I checked this with a simple for loop).  I am using epoxyModelGroup along with the controller similar to the epoxy sample app. 
[In this YouTube video, I show an example of the issue.](https://youtu.be/EyxwsWPc7v0)

**Comment by josephyanks** [comment]
This looks like a classic recycling view issue. How are you updating the data that those EditTexts represent? It seems like you're probably having those views recycled when you scroll back up top and aren't re-setting the state of the edit text. Some could would help narrow it down :)

**Comment by mradzinski** [comment]
It's indeed due to the recycling. Have you checked issue #426?

**Comment by nathanRamaNoodles** [comment]
Yeah, I checked that issue... And thanks to that, I am able to get the inputted data from the edit texts without any issue, however the weird cloning issue starts to happen again.

Basically, I am making a forum. So I get an arrayList from my Room database, and then I'm use epoxy to display the arrayList.  After the user fishes modifying the arrayList, I will update back to the room database.

**Comment by nathanRamaNoodles** [comment]
Code:

1.  [main fragment](https://pastebin.com/xCzqePKt)

2.  [Controller](https://pastebin.com/jabf8rzz)

3.  [Model Group](https://pastebin.com/i8yy8Cr0)

4. The hard part: [Edit Text Model](https://pastebin.com/g6hDCJUN)

**Comment by nathanRamaNoodles** [comment]
Anyone??

**Comment by nsekhar90** [comment]
Im facing the same issue. Can anyone help regarding this ? @nathanRamaNoodles how did you fix this ?
